.github/config/out_of_tree_extensions.cmake

```diff
@@ -93,7 +93,7 @@ if (NOT MINGW AND NOT ${WASM_ENABLED})
 #            ${LOAD_ICEBERG_TESTS} TODO: re-enable once autoloading test is fixed
```

Comment:
```suggestion
```

Cool, I might touch iceberg once more, but had not sent a PR yet, better to have this in, thanks!